web-client/public/js/update-app.js

```diff
@@ -15,6 +17,26 @@ const refreshApp = () => {
 };
 
 export const updateApp = grnState => {
+// Edge Weight Normalization
+    let valueValidator = (min, max, value) => {
+        return Math.min(max, Math.max(min, value));
+    };
+
+    let edgeWeightNormalizationInputValidation = (value) => {
+        return value === "" ? "" : valueValidator(MIN_EDGE_WEIGHT_NORMALIZATION, MAX_EDGE_WEIGHT_NORMALIZATION, value);
+    };
+
+    var validated = edgeWeightNormalizationInputValidation(value);
```

Comment:
@dondi I'm having a little bit of trouble figuring out why the code is breaking at this point on line 29, it seems that validated isn't a valid variable or something of that sort